submissions/ayoubkassi/GeeksChallenge.java

```diff
@@ -0,0 +1,92 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GeeksChallenge {
```

Comment:
Change the class name to Main as described in the readme

Reply:
@aboullaite Done